ui/README.md

```diff
@@ -0,0 +1,7 @@
+# Contributor Guide for UI component
+
+<p style="text-align: center"><em>If you'll like to go to the main Meshery Contributor guide [click here](../README.md).</em></p>
```

Comment:
Since your intentions are to direct the User to the `Meshery Contributor Guide`, It would be wise to use the direct link to the Contributor Guide instead of directing the user to the `readme.md` file itself. Correct me If I'm wrong.